tests/login.test.js

```diff
@@ -0,0 +1,88 @@
+describe('DigitalBank', () => {
+
+    //example test
+    it('Should enter the site', async () => {
+      browser.url('login');
+    });
+
+    it('Should log in to the demo account', async () => {
+      browser.url('login');
+      
+      let username = await $('[id="username"]');
+      await username.setValue('jsmith@demo.io');
+
+       let password = await $('[id="password"]');
+      await password.setValue('Demo123!');
+
+      let loginButton = $('[id="submit"]');
+      await loginButton.click()
```

Comment:
As noted in a previous test, in order to be complete, tests need an assertion clause (could use chai's **assertion** or **expect** function, but prefer **expect** because of readability reasons) to explicitly express what do you expect from the final state you leave the application after applying all those commands in order to signal that the right condition has been met.